compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/Draggable.kt

```diff
@@ -466,7 +467,9 @@ internal abstract class DragGestureNode(
         if (enabled && pointerInputNode == null) {
             pointerInputNode = delegate(initializePointerInputNode())
         }
-        pointerInputNode?.onPointerEvent(pointerEvent, pass, bounds)
+        if (pointerEvent.changes.fastAny { canDrag.invoke(it) }) {
```

Comment:
Should we do this patch in AOSP?

Reply:
It is preferable, yes, if @eymar has the setup ready. Otherwise it can be a part of the `ui` feature/bug upstreaming in the next 1-3 months.

Anyway, as it is critical for 1.7.0, we either can merge it now, or cherry-pick it from AOSP if we decide to upstream it right now.